packages/web-react/src/components/Tag/__tests__/Tag.test.tsx

```diff
@@ -1,21 +1,41 @@
-import React from 'react';
-import { render } from '@testing-library/react';
 import '@testing-library/jest-dom';
-import Tag from '../Tag';
+import { render } from '@testing-library/react';
+import React from 'react';
 import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+import { emotionColorPropsTest, sizeExtendedPropsTest } from '../../../../tests/providerTests/dictionaryPropsTest';
 import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import Tag from '../Tag';
 
 describe('Tag', () => {
   classNamePrefixProviderTest(Tag, 'Tag');
 
+  emotionColorPropsTest(Tag, 'Tag--');
+
+  sizeExtendedPropsTest(Tag);
+
   stylePropsTest(Tag);
 
   restPropsTest(Tag, 'span');
 
   it('should have default classname', () => {
     const dom = render(<Tag />);
 
-    expect(dom.container.firstChild).toHaveClass('Tag--default');
+    const element = dom.container.querySelector('span') as HTMLElement;
+    expect(element).toHaveClass('Tag--default');
+  });
+
+  it('should render text children', () => {
+    const dom = render(<Tag>Tag</Tag>);
+
+    const element = dom.container.querySelector('span') as HTMLElement;
+    expect(element.textContent).toBe('Tag');
+  });
+
+  it.each([['default']])('should render color %s', (color) => {
```

Comment:
There should not be the `each` for one value.

Reply:
add default deprecation, so now it makes sense :)